src/mds/MDCache.cc

```diff
@@ -8520,7 +8520,6 @@ int MDCache::path_traverse(const MDRequestRef& mdr, MDSContextFactory& cf,
 	  lov.add_rdlock(&dn->lock);
 	}
 	if (!mds->locker->acquire_locks(mdr, lov)) {
-	  dout(10) << "traverse: failed to rdlock " << dn->lock << " " << *dn << dendl;
 	  return 1;
```

Comment:
I understand that this is beyond the scope of this PR but are we sure we want to return `1`? This should be `CEPHFS_EPERM`

Reply:
errors are negative, so no?